server/controllers/notes.js

```diff
@@ -1,30 +1,59 @@
-const Notes = require('../models/notes')
+const Note = require('../models/notes');
 
 const notesController = {
     create: (req, res) => {
-        // code here
-        console.log(`create: ${req.body}`)
-        res.send('ok')
+        const note = new Note({
+            title: req.body.title,
+            notes: req.body.notes
+        });
+
+        note.save()
+            .then(note => {
+                return res.status(200).json({
+                    error: false,
+                    message: 'Note created',
+                    note: note
+                });
+            })
+            .catch(error => {
+                return res.status(500).json({
+                    error: true,
+                    message: 'Error Creating Note!'
+                });
+            });
     },
 
     update: (req, res) => {
-        console.log(`update: ${req.params}`)
-        res.send('ok')
+        console.log(`update: ${req.params}`);
+        res.send('ok');
     },
 
     delete: (req, res) => {
-        console.log(`delete: ${req.params}`)
-        res.send('ok')
+        console.log(`delete: ${req.params}`);
+        res.send('ok');
     },
 
     index: (req, res) => {
-       console.log(`index: ${req.body}`)
-       res.send('ok')
+        Note.find()
```

Comment:
Let's only return non-deleted notes.

Reply:
Will use the following `Note.find({ deleted: false })`